discovery_engine/test/ffi/types/init_config_test.dart

```diff
@@ -39,4 +39,26 @@ void main() {
     boxed.free();
     expect(res, equals(config));
   });
+
+  test('reading written init config ffi with ai config works', () {
+    final config = InitConfigFfi.fromParts(
+      apiKey: 'hjlsdfhjfdhjk',
+      apiBaseUrl: 'https://foo.example/api/v1',
+      feedMarkets: [
+        const FeedMarket(countryCode: 'DE', langCode: 'DE'),
```

Comment:
lang code should be lower case

Reply:
i just copy-pasted this from other tests, so i will also fix those.